spec/requests/users_spec.rb

```diff
@@ -0,0 +1,149 @@
+require "rails_helper"
```

Comment:
Minor observation: just realized that this is a regular Rails app and not an API one so writing feature specs with Capybara should be the way to go. Doesn't have to be done in this PR